lib/puppet/parser/functions/convert_to_ensurable.rb

```diff
@@ -0,0 +1,18 @@
+#
+# convert_to_ensurable.rb
+#
+# Simple converter function to map boolean values into their 'ensurable' counterparts, :present & :absent
+#
+module Puppet::Parser::Functions
```

Comment:
I think I'd be fine with bumping stdlib dependency to 4.10.0. stdlib's code is pretty stable, so updating that module is a low-risk operation.